src/components/ScenarioParameters/components/ScenarioParametersInputs/GenericTable.spec.js

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+
+import { _getColumnWithoutDepths } from './GenericTable';
+
+describe('Test if column parsing before send it to core works', () => {
```

Comment:
send it to core is not clear enough (it's better to describe receiving function instead of pointing the whole _cosmotech/core_ package)